docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -98,6 +98,38 @@ comprising the number of shards in the index, the total size of the index in
 bytes, and the maximum number of segments per shard in the index. Defaults to
 `false`, meaning that this information is omitted.
 
+`sort`::
+(Optional, string)
+Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorting by snapshot start time stamp.
++
+.Valid values for `sort`
+[%collapsible%open]
+====
+`start_time`::
+  Sort snapshots by their start time stamp and break ties by snapshot name.
+
+`duration`::
+  Sort snapshots by their duration and break ties by snapshot name.
+
+`name`::
+  Sort snapshots by their name.
+
+`indices`::
```

Comment:
Maybe `index_count` would be a better name than `indices`? Not that sorting by some amalgamation of the index names is going to be terribly useful, but that might be what people expect with the name `indices`.

Reply:
++ renamed